### ISSUE TITLE
loss.py 

### ISSUE THREAD

**Issue by bbkk5401** [ZiJianZhao/SeqGAN-PyTorch#11]
Hi, where is loss.py used?
I didn't see main.py import it .
you use nn.NLLLoss as loss function but loss.NLLLoss 
thanks

**Comment by studyhardstudyhard** [comment]
The two are the same
